backend/src/run_app.py

```diff
@@ -0,0 +1,12 @@
+if __name__ == "__main__":
+    import uvicorn
+
+    from src.core.configs import FAST_API_HOST, FAST_API_PORT
+
+    uvicorn.run(
+        "src.main:app",
+        host=FAST_API_HOST,
+        port=FAST_API_PORT,
+        log_level="debug",
+        reload=True,
+    )
```

Comment:
What is this ? We had generic system to run application in main.py, why did you create it ?

Reply:
this helps to fast create runner app to develop app with no needed environments and work fine in pycharm/vscode make project run instance and help to debug etc. We can remove it when develop process ends

Reply:
Okay